app/views/administrators/bussiness.html.erb

```diff
@@ -7,16 +7,16 @@
       <thead>
         <tr class="bg-gray-500 text-white text-left">
           <th class="w-52 py-2"> Nombre </th>
-          <th class="w-28"> Rol </th>
+          <th class="w-52 py-2"> Usuario </th>
           <th class="w-28"> Categoría </th>
           <th class="w-28"> Productos Totales </th>
         </tr>
       </thead>
       <tbody>
         <% @companies.each do |company| %>
           <tr>
-            <td class="py-4"><%= company.name %></td>           
-            <td class="py-4"></td>
+            <td class="py-4"><%= company.name %></td> 
+            <td class="py-4"><%= @companies.find(company.id).user.email %></td>
```

Comment:
I think you don't need to do the `find` here if the relation is `ok` you are going to be able to do something like this:
`company.user.email.`